test/EIP20CoGateway/getUtilityToken.js

```diff
@@ -0,0 +1,81 @@
+const chai = require('chai');
+const Web3 = require('web3');
+const sinon = require('sinon');
+const EIP20CoGateway = require('../../src/ContractInteract/EIP20CoGateway');
+const SpyAssert = require('../../test_utils/SpyAssert');
+
+const { assert } = chai;
+
+describe('EIP20CoGateway.getValueToken()', () => {
+  let web3;
+  let coGatewayAddress;
+  let coGateway;
+
+  let mockedUtilityTokenAddress;
+
+  let spyMethod;
+  let spyCall;
+
+  const setup = () => {
+    spyMethod = sinon.replace(
+      coGateway.contract.methods,
+      'utilityToken',
+      sinon.fake.returns(() => {
+        return Promise.resolve(mockedUtilityTokenAddress);
+      }),
+    );
+
+    spyCall = sinon.spy(coGateway, 'getUtilityToken');
+  };
+
+  const tearDown = () => {
+    sinon.restore();
+    spyCall.restore();
+  };
+
+  beforeEach(() => {
+    web3 = new Web3();
+    coGatewayAddress = '0x0000000000000000000000000000000000000002';
+    coGateway = new EIP20CoGateway(web3, coGatewayAddress);
+    mockedUtilityTokenAddress = '0x0000000000000000000000000000000000000003';
+  });
+
+  it('should return correct mocked utility token address', async () => {
```

Comment:
```suggestion
  it('should return correct utility token address', async () => {
```
The test should assert that the class actually returns the correct address, if you know what I mean.